packages/cli/doc-plugin-typedoc/src/index.ts

```diff
@@ -0,0 +1,83 @@
+import path from 'path';
+import { Application, TSConfigReader } from 'typedoc';
+import { API_DIR } from './constants';
+import { resolveSidebar } from './sidebar';
+
+export interface PluginTypeDocOptions {
+  /**
+   * The entry points of modules.
+   * @default []
+   */
+  entryPoints: string[];
+  /**
+   * The output directory.
+   * @default 'api'
+   */
+  outDir?: string;
+}
+
+export function pluginTypeDoc(options: PluginTypeDocOptions) {
+  let docRoot: string | undefined;
+  const { entryPoints = [], outDir = API_DIR } = options;
+  return {
+    name: 'doc-plugin-typedoc',
+    async addPages() {
+      return [
+        {
+          routePath: `${outDir.replace(/\/$/, '')}/`,
+          filepath: path.join(docRoot!, outDir, 'README.md'),
+        },
+      ];
+    },
+    // The module tools will build error when use the types of doc-core
```

Comment:
If module tools build failed due to the type, we should dig and fix it, @10Derozan cc